config/deploy/stage.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+server 'was-registrar-app-stage.stanford.edu', user: 'was', roles: %w[web app db]
+
+Capistrano::OneTimeKey.generate_one_time_key!
+set :rails_env, 'staging'
```

Comment:
AFAICT, we @sul-dlss/infrastructure-team folks have been inconsistent about this, but my hunch is that we should use `'production'` in all deployed environments since it's a built-in Rails environment. I've been bitten in the past by inventing new Rails environments. Does anyone strongly feel we should use `'staging'` instead of `'production'` here and, if so, can you help me see the light?

Reply:
Btw, love to see us using circle instead of travis now!

Reply:
Yup, got bit (i think) by that and have changed it. I'm having related (I think) issues with assets and sprockets (which we shouldn't be using) and may need to hand this off before I'm done for the week. So any insight there would be helpful.

Reply:
also - it's "stage" not "staging".   We try to be consistent and use "stage" for the VMs and for the capistrano ... targets or whatever they are called.

Reply:
@ndushay We are pretty consistent in terms of VM names and Capistrano stages (yes, super confusing that they are called stages and we use "stage" to mean "staging...").

We are less consistent in terms of Rails environments. I see a mix of `stage.{rb,yml}` and `staging.{rb,yml}` in `shared_configs`. There are more that use `staging` than `stage` by a rough count. (argo-stage, dor-indexing-app-stage, kurma-robots1-stage, etc.).

By this same convention, we should prefer `prod` to `production` but we don't do that with Rails environments (we do with VM names and Capistrano stages of course...).

I'd like to see us move away from using custom Rails environments, so it's good to see we don't have too many codebases doing this.